### ISSUE TITLE
DW10: Order states visualization

### ISSUE THREAD

**Issue by Wassini** [dynamicweb/DynamicWeb#9]
When adding additional states to an order flow, this is shown on the order, but it seems that a complete order have gone through all states:

![image](https://github.com/dynamicweb/DynamicWeb/assets/17749701/28d6f424-935f-49cd-a94b-6c3c5b8bd26b)

Most order would go from new to complete.

**Comment by DynamicRas** [comment]
Hi, 

This is because it's a stepped flow. What you can do is to sort the states so Completed is before Error and Resending. Would that work for you?

BR Rasmus Sanggaard

**Comment by nicped** [comment]
This is not a feature request and belongs elsewhere.

**Comment by Wassini** [comment]
This was just suggestion to enhance the status flow visualization to accurately represent the path a sales order has followed. 
This is neither a bug report nor a request for operational guidance.

**Comment by DynamicRas** [comment]
@Wassini Agree. Let's keep it open and see if there is a general need for a better way of showing the states on the order overview.
